### ISSUE TITLE
Display the App Id on the App Settings page

### ISSUE THREAD

**Issue by dfiggtns** [microsoft/appcenter#1122]
**Describe the solution you'd like**
It would be very useful have the App Id (the one used in the url) shown on the App Settings page, as well as the App's Name.

**Describe alternatives you've considered**
N/A.

**Additional context**
While trying to set up the fastlane-plugin-appcenter plugin, I kept getting the message when I tried to upload the ios version of my app:

    Platform does not match. Expected value: Android. Actual value: iOS

It turned out that I needed to use the app id that appears in the url, rather than the app name that appears in the UI.

What would be very useful would be to have the app id shown on the App Settings page, as well as the app name.

**Comment by ScottArbeit** [comment]
Hi @dfiggtns!

Thank you the suggestion. Over the next many months, one of the things we're hoping to do is a little bit of refactoring on this part of our UI, and I'd like to make the experience around App Name vs. App Id (or URL slug) more clear.

I joined the team recently, and a couple of weeks ago I ran into the same confusion as you did while writing my first app using our REST API. It's a paper cut that we can clean up.

I'm going to keep this issue open and tag it so we follow up on it when we do that work.
